streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -56,7 +56,7 @@ public String name() {
     }
 
     /**
-     * Set the window emit duration in milliseconds of system time
+     * Set the window emit duration in milliseconds of system time.
      */
     public Windows emit(long duration) {
```

Comment:
We should perhaps rename the parameter `duration` to `durationMs`, which helps users to remember what the time unit is (until we perhaps have support for `TimeUnit` directly).